internal/common/labels.go

```diff
@@ -47,6 +48,20 @@ func AddAppLabels(requestInstance *ssp.SSP, name string, component AppComponent,
 	return obj
 }
 
+// AddAppLabels to the provided object metadata
+// Name will translate into the AppKubernetesNameLabel
+// Component will translate into the AppKubernetesComponentLabel
+// Instance wide labels will be taken from the request if available
+func AddAppLabelsToMeta(objectMeta metav1.ObjectMeta, requestInstance *ssp.SSP, name string, component AppComponent) metav1.ObjectMeta {
```

Comment:
Be careful with passing around copies of `objectMeta`. The original `AddAppLabels` is working slightly differently, we're the original object is modified. This func takes a copy of `objectMeta` and returns the modified copy instead.

Can you make both `AddAppLabels` and `AddAppLabelsToMeta` use a helper func to do the acutal work between L42-L46 and L56-60, which is identical? (dedup this block)

Reply:
Done. Thanks!